test/functional/collection.test.js

```diff
@@ -548,7 +548,7 @@ describe('Collection', function() {
       const ObjectID = configuration.require.ObjectID;
       db.createCollection('save_error_on_save_test', (err, collection) => {
         // Create unique index for username
-        collection.createIndex([['username', 1]], configuration.writeConcernMax(), err => {
+        collection.createIndex({ username: 1 }, configuration.writeConcernMax(), err => {
```

Comment:
This test broke, is `createIndex` supposed to support that array syntax?

Reply:
god I have no idea what that syntax is, but I think we need to support it :/